docs/how_to_contribute.rst

```diff
@@ -0,0 +1,62 @@
+
+Contributor Guide
+=================
+
+Step-by-step
+------------
+This guide will take you through the necessary steps in order to contribute code to the repository.
+
+
+1. Setup
+^^^^^^^^^^^^^^^^^^
+Fork the main repository, and clone your fork onto the machine of your choice.
+
+Make sure to install and activate the environment with::
+
+   conda create -n gcm-filters-env -c conda-forge --file requirements.txt --file requirements-dev.txt
+
+If you find that `conda` takes a long time, try to install `mamba` with ``conda install mamba`` and then
+and do::
+
+   conda create -n gcm-filters-env -c conda-forge --file requirements.txt --file requirements-dev.txt
+
+
+and finally activate the environment::
+
+   conda activate gcm-filters-env
+
+Before you can efficiently test your code you need to install this package in the now activated environment::
+
+   pip install -e . --no-deps
+
+For the linting also install `pre-commit <https://pre-commit.com>`_ now::
+
+   pre-commit install
+
+2. Write your code
+^^^^^^^^^^^^^^^^^^
+This one is pretty obvious 😁
+
+3. Write and run tests
+^^^^^^^^^^^^^^^^^^^^^^
+
+Now figure out a good way to test the functionality of your code and run the test suite with::
+
+   pytest
```

Comment:
```suggestion
   py.test -v
```

Reply:
whats the advantage of py.test? `-v should be defined in the config files?`

Reply:
It is not actually. Let me take care of that.

Reply:
See #17

Reply:
The [pytest docs](https://docs.pytest.org/en/stable/) use `pytest` instead of `py.test`, so I would opt to keep it that way?